austin_parham/func_intermediate_1.py

```diff
@@ -0,0 +1,48 @@
+def randInt():
+	import random
```

Comment:
we only need to import random once and it should be on the first line of the file.
https://www.python.org/dev/peps/pep-0008/#imports